content/docs/command-reference/exp/remove.md

```diff
@@ -125,3 +125,24 @@ $ dvc exp list myremote
 master:
         conic-ease
 ```
+
+Conversely, to keep only specific experiments (and remove all others), give their names to `dvc exp remove` with the `--keep` flag :
```

Comment:
You would need to run linters most likey

Reply:
yep, thanks probably this should have some instruction on how to setup the project https://dvc.org/doc/contributing/docs

Reply:
Thanks ! That helped a lot !